content/docs/distribution/outbound_shipments.md

```diff
@@ -390,7 +390,7 @@ However you can still manually choose those lines and issue the expired stock!
 
 ![pack sizes warning!](/docs/distribution/images/os_warning_pack_sizes.png)
 
-This warning tells you that because of the available pack sizes, the system has rounded up the quantity requested. In this example, there is no pack size that is less than `12`. There are packs of `20` available though, so when 12 tablets are requested, the system has rounded up the request to `2` packs of `20`.
+This warning tells you that because of the available pack sizes, the system has rounded up the quantity requested. In this example, there is no pack size that is less than `5`. There are packs of `10` available though, so when252 tablets are requested, the system has rounded up the request to `1` pack of `10`.
 
 ![pack sizes warning!](/docs/distribution/images/os_warning_no_quantity.png)
 
```

Comment:
Ugh can't comment on it but line below this one really needs a comma

`If you add an item, forget to enter a quantity to issue,`